chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/extra/JobResponseEmail.java

```diff
@@ -0,0 +1,95 @@
+package gov.healthit.chpl.scheduler.job.extra;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.Logger;
+import org.springframework.core.env.Environment;
+
+import gov.healthit.chpl.util.EmailBuilder;
+
+public class JobResponseEmail {
+    private static final String FILENAME = "jobResponse.csv";
+
+    public void sendEmail(final Environment env, final String emailAddress,
+            final List<JobResponseTriggerWrapper> triggerWrappers, final Logger logger) {
+        try {
+            new EmailBuilder(env)
+                    .recipients(new ArrayList<String>(Arrays.asList(emailAddress)))
+                    .subject("Retire 2014 Listing Job Report")
```

Comment:
The subject seems a little specific for a class that looks like it could otherwise be re-used. Can you make it a property or passed in from the job or something? I think for the other jobs that send an email the subject is in a property.